src/App.js

```diff
@@ -1,25 +1,31 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
+  const buttonsInfo = [
+    { text: 'AC', changeBg: false },
+    { text: '+/-', changeBg: false },
+    { text: '%', changeBg: false },
+    { text: '/', changeBg: true },
```

Comment:
- [ ] The calculator should match the design. In keeping with that this button should be &divide;. If you do not have the button on your PC consider using the HTML entity code **`&divide;`**